contracts/minting/MintController.sol

```diff
@@ -28,12 +28,17 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 
 /**
  * @title MintController
- * @dev allows control of configure/remove minter by different addresses
- *
+ * @dev Implementation of the abstract Controller contract, in which
```

Comment:
@notice The MintController contract manages minters for a contract that implements the MinterManagerInterface.  It lets the owner designate certain addresses as controllers, and these controllers then manage the minters by adding and removing minters, as well as modifying their minting allowance.  A controller may manage exactly one minter, but the same minter address may be managed by multiple controllers.
@dev MintController inherits from the Controller contract.  It treats the Controller workers as minters.